datasets/eurlex/README.md

```diff
@@ -0,0 +1,232 @@
+  "eurovoc_concepts": ["192", "2356", "2560", "862", "863"]
+}
+```
+
+### Data Fields
+
+The following data fields are provided for documents (`train`, `dev`, `test`):
+
+`celex_id`: The official ID of the document. The CELEX number is the unique identifier for all publications in both Eur-Lex and CELLAR.\
+`document_type`: The type of the document (e.g., Directive, Regulation, Decision)\
+`title`: The title of the document.\
+`header`: The first part of the document, which includes the title and name of the legal body enforcing the legal act.\
+`recitals`: The second part of the document, which contains legal background references.\
+`main_body`: The main part of the document, organized in articles or even bigger units (e.g., chapters, sections).\
+`eurovoc_concepts`: The relevant EUROVOC concepts (labels).
+
+For simplicity, we provide the full content (`text`) of each document, which is represented by its `header`, `recitals` and `main_body`.
+
+If you want to use the descriptors of EUROVOC concepts, similar to Chalkidis et al. (2020), please load: https://archive.org/download/EURLEX57K/eurovoc_concepts.jsonl
```

Comment:
They are already included in the dataset loaded with `load_dataset` no ?

Reply:
I updated README to represent the actual objects.